appengine/headless-chrome/app.js

```diff
@@ -35,7 +35,7 @@ app.use(async (req, res) => {
   const page = await browser.newPage();
   await page.goto(url);
   const imageBuffer = await page.screenshot();
-  await browser.close();
+  browser.close();
```

Comment:
would this have made more sense to put the res.set & res.send calls before `await browser.close`? Keeping the await ensured that the browser session was closed without error before the route was finished.

Albeit there is no error handling in this example, so any throw would cause an unhandled exception error, but it does seem odd to me to have this async operation continue doing it's thing after the function has returned